content/logs/languages/php.md

```diff
@@ -0,0 +1,562 @@
+    ##   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    ##   tags: (optional) add tags to each logs collected
+
+  - type: file
+    path: /path/to/your/php/application-json.log
+    service: php
+    source: php
+    sourcecategory: sourcecode
+```
+
+### Add meta field and context
+
+Much of the useful information comes from additional context data added to your logs and events.
+Monolog makes this convenient by providing methods for setting thread-local context data that is then submitted automatically with all events. At any moment, log an event with contextual data:
+
+```php
+$logger->info('Adding a new user', array('username' => 'Seldaek'));
+```
+
+Monolog comes with a pre-processor feature. It's a simple callback that enriches your events with metadata you can set (e.g., the session id, the request id, etc.):
+
+```php
+ <?php 
+    
+$log->pushProcessor(function ($record) {
+
+    // record the current user
+    $user = Acme::getCurrentUser();
+    $record['context']['user'] = array(
+        'name' => $user->getName(),
+        'username' => $user->getUsername(),
+        'email' => $user->getEmail(),
+    );
+
+    // Add various tags
+    $record['ddtags'] = array('key' => 'value');
+
+    // Add various generic context
+    $record['extra']['key'] = 'value';
+
+    return $record;
+});
+```
+
+### Framework integration 
+
+Monolog is a part of the following frameworks:
+
+* [Symfony2, Symfony3](/logs/languages/php/#symfony-v2-v3)
+* [PPI](/logs/languages/php/#ppi)
+* [Laravel 4 & 5](/logs/languages/php/#laravel)
+* [Silex](/logs/languages/php/#silex)
+* [Lumen](/logs/languages/php/#lumen)
+* [CakePHP](/logs/languages/php/#cakephp)
+
+Integrate Monolog with your framework then configure your logger: 
+ 
+```php
+ <?php
+// Check if the Monolog library is well loaded
+//use Monolog\Logger;
+//use Monolog\Handler\StreamHandler;
+//use Monolog\Formatter\JsonFormatter;
+  
+// with the monolog instance
+$monolog = ...
+  
+///// Log shipper configuration
+
+$formatter = new JsonFormatter();
+$stream = new StreamHandler(__DIR__.'/application-json.log', Logger::DEBUG);
+$stream->setFormatter($formatter);
+
+$monolog->pushHandler($stream);
+return $r;
+```
+
+### Configure your Datadog agent
+
+Create a `php.d/conf.yaml` file in your `conf.d/` folder with the following content:
+
+```yaml
+init_config:
+
+instances:
+    
+##Log section
+logs:
+
+    ## - type : file (mandatory) type of log input source (tcp / udp / file)
+    ##   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
+    ##   service : (mandatory) name of the service owning the log
+    ##   source : (mandatory) attribute that defines which integration is sending the logs
+    ##   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    ##   tags: (optional) add tags to each logs collected
```

Comment:
[Same comment](https://github.com/DataDog/documentation/pull/2127/files#r175451646) here about the colon spacing.